x/pubkey/keeper/endblock.go

```diff
@@ -141,12 +141,14 @@ func (k Keeper) JailValidators(ctx sdk.Context, keyIndex utils.SEDAKeyIndex) err
 			if err != nil {
 				return err
 			}
-			err = k.slashingKeeper.Jail(ctx, consAddr)
-			if err != nil {
-				return err
+			if !val.IsJailed() {
+				err = k.slashingKeeper.Jail(ctx, consAddr)
+				if err != nil {
+					return err
+				}
```

Comment:
I think we can add a test case for this right?

Reply:
Added an already jailed validator to the test [b9e5d32](https://github.com/sedaprotocol/seda-chain/pull/522/commits/b9e5d32f768f3fb03b3347fa4e75b3bcd8867451)